README.md

```diff
@@ -120,9 +120,9 @@ shell with:
 > python x.py build
 ```
 
-Currently building Rust only works with some known versions of Visual Studio. If
+Currently, building Rust only works with some known versions of Visual Studio. If
 you have a more recent version installed the build system doesn't understand
-then you may need to force rustbuild to use an older version. This can be done
+then you may need to force rust build to use an older version. This can be done
```

Comment:
rustbuild is correct, it's the name of the build tool that bootstraps the compiler and builds everything with Cargo

Reply:
@jonas-schievink  Thanks for correction. I have reverted it.